pom.xml

```diff
@@ -32,10 +32,9 @@
 	</licenses>
 	<dependencies>
 		<dependency>
-			<groupId>com.codahale.metrics</groupId>
+			<groupId>io.dropwizard.metrics</groupId>
 			<artifactId>metrics-core</artifactId>
-			<version>3.0.1</version>
-			<scope>provided</scope>
+			<version>3.1.0</version>
```

Comment:
why do you remove the "provided" scope ?
I don't want to force the version and groupId of metrics.